src/lib/functions/server.js

```diff
@@ -107,8 +160,10 @@ const createHandler = function ({ config, functionsRegistry }) {
     }
 
     if (config && config.authlify && config.authlify.authlifyTokenId != null) {
-      const { authlifyTokenId, netlifyToken, siteId } = config.authlify
-      event.authlifyToken = generateAuthlifyJWT(netlifyToken, authlifyTokenId, siteId)
+      // XXX(anmonteiro): this name is deprecated. Delete after 3/31/2022
+      const jwt = generateNetlifyGraphJWT(config.netlifyGraphConfig)
+      event.authlifyToken = jwt
+      event.netlifyGraphToken = jwt
```

Comment:
We should get the changes ported to `nf-server` and `proxy` soon as well

Reply:
Similarly, anywhere we inject `ONEGRAPH_AUTHLIFY_TOKEN` in the env we should also inject `NETLIFY_GRAPH_TOKEN` so we're future-proof for the public launch.